charts/nautobot/values.yaml

```
@@ -254,6 +254,23 @@ nautobot:
     #   matchLabel:
     #     nautobot-storage: static
 
+  # Add a Persistent Volume Claim for Nautobot media files
+  persistenceMediaFiles:
+    # --- Enable PVC and the relevant volumes
+    enabled: true
```

I think it would be good to disable this feature by default.
```suggestion
    enabled: false
```